starknet_py/net/schemas/gateway.py

```diff
@@ -386,6 +390,68 @@ def make_dataclass(self, data, **kwargs) -> ContractClass:
         return ContractClass(**data)
 
 
+class NewEntryPointSchema(Schema):
+    function_idx = fields.Integer(data_key="function_idx", required=True)
+    selector = Felt(data_key="selector", required=True)
+
+    @post_load
+    def make_dataclass(self, data, **kwargs) -> NewEntryPoint:
+        return NewEntryPoint(**data)
+
+
+class NewEntryPointsByTypeSchema(Schema):
+    constructor = fields.List(
+        fields.Nested(NewEntryPointSchema()), data_key="CONSTRUCTOR", required=True
+    )
+    external = fields.List(
+        fields.Nested(NewEntryPointSchema()), data_key="EXTERNAL", required=True
+    )
+    l1_handler = fields.List(
+        fields.Nested(NewEntryPointSchema()), data_key="L1_HANDLER", required=True
+    )
+
+    @post_load
+    def make_dataclass(self, data, **kwargs) -> NewEntryPointsByType:
+        return NewEntryPointsByType(**data)
+
+
+class NewContractClassSchema(Schema):
+    contract_class_version = fields.String(
+        data_key="contract_class_version", required=True
+    )
+    sierra_program = fields.List(
+        fields.String(),
```

Comment:
I think we have a `Felt` field, could it be used here?

Reply:
I though about it but it doesn't seem useful. If anything, I'd assume that anyone trying to use sierra program bytecode returned from the network would expect it to be just in the format it came from the network.